spec/unit/crypto.spec.ts

```diff
@@ -451,8 +454,10 @@ describe("Crypto", function() {
             // cancelAndResend will call sendToDevice twice:
             // the first call to sendToDevice will be the cancellation
             // the second call to sendToDevice will be the key request
-            expect(aliceClient.sendToDevice).toBeCalledTimes(3);
-            expect(aliceClient.sendToDevice.mock.calls[2][2]).not.toBe(txnId);
+            expect(aliceSendToDevice).toBeCalledTimes(3);
+            expect(aliceSendToDevice.mock.calls[2][2]).not.toBe(txnId);
+
+            jest.useRealTimers();
```

Comment:
I'd suggest putting this in the afterEach - it doesn't matter if it's called after some tests where it doesn't need to be, but if tests throw exceptions then this would need to be in a try/finally, so afterEach is tidier.